quickwit/quickwit-proto/protos/quickwit/search.proto

```diff
@@ -165,6 +165,11 @@ message SearchRequest {
   // that will make it possible to paginate through the results
   // in a consistent manner.
   optional uint32 scroll_ttl_secs = 15;
+
+  // Document with sort tuple smaller or equal to this are discarded to
+  // enable pagination.
+  // If split_id is empty, no comparison with _shard_doc should be done
```

Comment:
What is the purpose of making `split_id` empty?

Reply:
if you sort by `["my_field1", "my_field2"]`, without a final `_shard_doc`, ES doesn't return a unique doc identifier, so when the client comes back for more, it can't provide such an identifier in `search_after`. That's when `split_id` is empty